python/tank/constants.py

```diff
@@ -32,6 +32,9 @@
 # the storage name that is treated to be the primary storage for tank
 PRIMARY_STORAGE_NAME = "primary"
 
+# hook to modify default storage root based on project root
+MODIFY_DEFAULT_STORAGE_ROOT_HOOK_NAME = "modify_default_storage_root"
```

Comment:
Why did we go for this name and not just "storage_root" or "default_storage_root"?